app-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Could you add `python manage.py db stamp head` before the other two commands? I can't explain exactly why but sometimes alembic has trouble figuring out the current state of the database if you don't run that.

Reply:
Hey @egiurleo would this cause any problems with creating new migrations? I'm not very familiar with alembic/stamp so wondering if stamping the head would cause any issues.

Reply:
Nope! This is what we run on Circle CI :)

Reply:
Sweet! Added.

Reply:
Readme also updated - I'll give this a bit of time just in case there is something else you want to add but I'll probably merge this in today finally :)